packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -3,22 +3,24 @@
  */
 import { get } from 'lodash';
 
+/** @typedef {Record<string, import('./reducer').State>} State */
```

Comment:
Forgot to answer in the previous PR, so I'm going to do it here — what is the reason behind creating the `EquivalentKeyMap` class/type, instead of using a regular `Map`?

Reply:
We need to declare that the EquivalentKeyMap class has some kind of API. It happens to have the same API as Map but is different in implementation: https://github.com/aduth/equivalent-key-map

I'm not sure I totally understand what you're proposing as an alternative though. EquivalentKeyMap module exports a specific class which we're using, it's not a Map, just has the same API as Map.

Reply:
I was not proposing an alternative necessarily, I guess I was just confused by the fact that we used `Map` types to type `EquivalentKeyMap`, but now things are clear to me :) all good from my side!